redis/src/parser.rs

```diff
@@ -351,6 +494,79 @@ mod tests {
         assert_eq!(codec.decode_eof(&mut bytes), Ok(None));
     }
 
+    #[test]
+    fn decode_resp3_double() {
+        let val = parse_redis_value(b",1.23\r\n").unwrap();
```

Comment:
Could we add a couple additional tests here, e.g., scientific notation / infinity, just for my paranoia's sake? :sweat_smile:

Reply:
also NaN / -NaN, please.

Reply:
@nihohit can u update #749 for scientific E notation ?